packages/server/__tests__/arpa_reporter/server/helpers/with-tenant-id.js

```diff
@@ -8,12 +8,12 @@ const { requestProviderMiddleware } = require('../../../../src/arpa_reporter/use
  * @returns {Promise}
  * A promise that resolves to the return value of the callback, if any.
  */
-async function withTenantId(tenantId, callback) {
+async function withTenantId(tenantId, callback, ...args) {
```

Comment:
ah, thank you! 
I wrote this code mainly because I was initially doing this:`await withTenantId(TENANT_A, findRecipient('name', 'IAA'));` which was running into a TypeError since I was passing in the response of the function-call rather than the function itself. This is now updated.